### DISCUSSION TITLE
Sync from MyWhoosh Site

### DISCUSSION THREAD

**Discussion by urinud** [JayQueue/MyWhoosh2Garmin#4] [Ideas]
The current script depends on computer stored fit files.
Fit files can be also downloaded from MyWhoosh Site.
By syncing from web can be better automated and also works for non-desktop versions of MyWhoosh.

**Comment by JayQueue** [comment]
True, but I wanted something quick to use. 
But I'm going to refactor the whole thing during the holidays so it's using the downloaded fit file.

Thanks for the feedback

**Reply by JayQueue** [reply]
I was working on it and then I thought about the Terms and conditions and I found this
![image](https://github.com/user-attachments/assets/e0a45887-90ce-4e7e-a878-a306975ca24d)

So I'm skipping this. Sorry @urinud

**Reply by urinud** [reply]
That's ok, good that you catch it.
What do you think trying to get explicit permission from [MyWhoosh](https://www.mywhoosh.com/contact-us/)?

**Reply by JayQueue** [reply]
@urinud I emailed them, explaining what the goal is and still waiting for a response.

**Reply by JayQueue** [reply]
Sooo... 
![image](https://github.com/user-attachments/assets/eb08e664-0645-4818-839f-8c8d40041f7a)
